--- flac/configure.ac.org	2013-05-27 10:08:57.000000000 +0200
+++ flac/configure.ac	2016-12-15 12:16:16.947514051 +0100
@@ -392,6 +392,9 @@
 		fi
 	fi
 
+	if test "x$GCC_MAJOR_VERSION$GCC_MINOR_VERSION" = "x47" ; then
+		XIPH_ADD_CFLAGS([-fno-inline-small-functions])
+		fi
 
 XIPH_ADD_CFLAGS([-Wextra])
 XIPH_ADD_CFLAGS([-Wdeclaration-after-statement])
--- flac/configure.org	2013-05-27 10:11:54.740045571 +0200
+++ flac/configure	2016-12-15 12:08:33.979735735 +0100
@@ -19879,6 +19879,39 @@
     conftest$ac_exeext conftest.$ac_ext
 
 		fi
+
+	if test "x$GCC_MAJOR_VERSION$GCC_MINOR_VERSION" = "x47" ; then
+		{ $as_echo "$as_me:${as_lineno-$LINENO}: checking if $CC accepts -fno-inline-small-functions" >&5
+$as_echo_n "checking if $CC accepts -fno-inline-small-functions... " >&6; }
+	ac_add_cflags__old_cflags="$CFLAGS"
+	CFLAGS="-fno-inline-small-functions"
+	cat confdefs.h - <<_ACEOF >conftest.$ac_ext
+/* end confdefs.h.  */
+
+			#include <stdio.h>
+
+int
+main ()
+{
+puts("Hello, World!"); return 0;
+  ;
+  return 0;
+}
+_ACEOF
+if ac_fn_c_try_link "$LINENO"; then :
+  { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
+$as_echo "yes" >&6; }
+			CFLAGS="$ac_add_cflags__old_cflags -fno-inline-small-functions"
+else
+  { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
+$as_echo "no" >&6; }
+			CFLAGS="$ac_add_cflags__old_cflags"
+
+fi
+rm -f core conftest.err conftest.$ac_objext \
+    conftest$ac_exeext conftest.$ac_ext
+
+		fi
 	fi
 
 
